### PR TITLE
added recipe for pycontour

### PR DESCRIPTION
**Pull request by sarthakpati**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sarthakpati** [comment]
@conda-forge/help-python this is ready for merge. Thank you!

**Comment by beckermr** [comment]
Linux 64 build is failing.

**Comment by sarthakpati** [comment]
> Linux 64 build is failing.

I am unable to figure out why [this error](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=301580&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=726) is coming up for Linux but not for macOS or Windows:
```bash
ImportError: libGL.so.1: cannot open shared object file: No such file or directory
```

A quick search landed me on [this answer](https://stackoverflow.com/a/66473309/1228757):
```bash
apt-get update && apt-get install -y python3-opencv
pip install opencv-python
```

But I assume that should not really be needed?

**Comment by sarthakpati** [comment]
@conda-forge/help-python this is ready for merge. Thank you!

File: recipes/pycontour/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (dopplershift) [inline recipes/pycontour/LICENSE:1]: This file is already included in the PyPI tarball, so you can delete it from this PR.

**Review by dopplershift — CHANGES_REQUESTED** [review]
This looks good, just need to remove the unneeded license file.

File: recipes/pycontour/LICENSE (hunk 1 above)
Comment (sarthakpati) [reply recipes/pycontour/LICENSE:1]: Done.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**